src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
-package core.basesyntax;
-
-public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
-    }
-}
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
+        return "" + Color.values()[index];

```

Comment:
```suggestion
        return Color.values()[index].name();
```
It’s  hardcode, just call method `name()`